lib/active_merchant/billing/gateways/stripe_payment_intents.rb

```diff
@@ -207,10 +207,10 @@ def setup_future_usage(post, options = {})
       end
 
       def add_connected_account(post, options = {})
-        return unless transfer_data = options[:transfer_data]
```

Comment:
Sorry if this was already discussed elsewhere, but wouldn't this be a backwards-incompatible change for other users of this gem? What's the driving need for us to change the option keys here?

Reply:
I noticed as I was about to add this as a GSF that it seems pretty rare for other gateways to have a hash as an option for something pretty simple like this set of fields, so I thought it might be good to simplify it. It _seems_ low risk since I just added this gateway within the last 2 weeks or so, but you're right that this would be backwards incompatible. To be safe, I can close this out and just handle on our end though in case anyone is already using this if you all think that's the better approach!

Reply:
Ah, didn't realize it was a change to that recent an addition. If there hadn't been a gem release yet for that code I would say we're probably safe, but if it had already been released with that as part of the public API we'd probably want to err on the safe side.

Reply:
Yeah, doublechecked, and the gateway addition is at the very top of the changelog HEAD, so doesn't look like it's been released in a version yet